freebridgefe/src/stores/chatStore.ts

```
@@ -32,112 +37,156 @@ export const useChatStore = defineStore('chat', () => {
+            subscriptions[roomId].unsubscribe();
+            delete subscriptions[roomId];
+        }
+    }
 
-    // Getters
+    function disconnectWebSocket() {
+        if (stompClient) {
+            stompClient.deactivate();
+            stompClient = null;
+        }
+        Object.keys(subscriptions).forEach((id) => unsubscribeFromRoom(id));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)